edgedb/protocol/protocol.pyx

```diff
@@ -1089,22 +1089,15 @@ cdef class SansIOProtocol:
 
         in_dc_type = type(in_dc)
 
-        if in_dc_type in {NullCodec, EmptyTupleCodec}:
-            # TODO: drop EmptyTupleCodec when 1.0 RC1 is released.
```

Comment:
I am not quite sure if that means removing the codec only from this condition or removing the codec completely

Reply:
We can address dropping `EmptyTupleCodec` itself in a separate PR - it shouldn't be an issue keeping it I think.